docs/user/dashboard/vega-reference.asciidoc

```diff
@@ -248,7 +248,7 @@ url: {
 // The result is a geojson file, get its features to use
 // this data source with the "shape" marks
```

Comment:
The most commonly used EMS layers are published as TopoJSON files. But some of the legacy layers are still only available as GeoJSON files. Unfortunately, we don't currently have an easy way for a user to determine whether or not their chosen layer is published as TopoJSON or GeoJSON. Once we resolve https://github.com/elastic/ems-landing-page/issues/430 we can update the docs to direct the user to find the default format type for their layer at https://maps.elastic.co and set the `format` accordingly. cc @jsanz

```suggestion
// The result is either a topojson file or a geojson file. 
// Get its features to use this data source with the "shape" marks
```